src/RA_MemManager.h

```diff
@@ -2,22 +2,12 @@
 
 #include "RA_Condition.h"
 
-class MemCandidate
+struct MemCandidate
 {
-public:
-    MemCandidate()
-        : m_nAddr(0),
-        m_nLastKnownValue(0),
-        m_bUpperNibble(FALSE),
-        m_bHasChanged(FALSE)
-    {
-    }
-
-public:
-    unsigned int m_nAddr;
-    unsigned int m_nLastKnownValue;		//	A Candidate MAY be a 32-bit candidate!
-    bool m_bUpperNibble;				//	Used only for 4-bit comparisons
-    bool m_bHasChanged;
+    unsigned int m_nAddr{ 0U };
+    unsigned int m_nLastKnownValue{ 0U };		//	A Candidate MAY be a 32-bit candidate!
+    bool m_bUpperNibble{ false };				//	Used only for 4-bit comparisons
+    bool m_bHasChanged{ false };
```

Comment:
These had narrowing problems, `BOOL `is a `typedef `of `int `(4 bytes on windows) `bool `is 1 byte everywhere.